### PR TITLE
couchdb: update to 3.0.1

### PR DESCRIPTION
**Pull request by arete**

### CODE REVIEW

File: srcpkgs/couchdb/template [hunk 1]
```diff
@@ -1,40 +1,111 @@
...
 
 system_accounts="couchdb"
 couchdb_homedir="/var/lib/couchdb"
+couchdb_shell="/bin/sh"
+couchdb_descr="CouchDB Administrator"
 make_dirs="
-	/var/lib/couchdb 0700 couchdb couchdb
-	/var/log/couchdb 0750 couchdb couchdb"
+ /var/lib/couchdb 0770 couchdb couchdb
+ /var/log/couchdb 0750 couchdb couchdb"
```
Comment (Johnnynator) [inline srcpkgs/couchdb/template:27]: Why `0770` for `lib` and `0750` for var?

File: srcpkgs/couchdb/template [hunk 2]
```diff
@@ -1,40 +1,111 @@
...
-version=1.7.1
-revision=2
+version=3.0.1
+revision=1
 wrksrc="apache-couchdb-${version}"
-build_style=gnu-configure
-conf_files="/etc/couchdb/default.ini /etc/couchdb/local.ini"
-hostmakedepends="erlang pkg-config"
-makedepends="libressl-devel icu-devel js-devel libcurl-devel"
+build_style=gnu-makefile
```
Comment (Johnnynator) [inline srcpkgs/couchdb/template:6]: Why change to gnu-makefile if you call `./configure` later? if it is not a `gnu-configure` than you can use `build_style=configure`

File: srcpkgs/couchdb/template (hunk 1 above)
Comment (arete) [reply srcpkgs/couchdb/template:27]: I have update the pull request

File: srcpkgs/couchdb/template (hunk 2 above)
Comment (arete) [reply srcpkgs/couchdb/template]: i have update te pull request

**Comment by arete** [comment]
this pull request not build with master after update erlang #22108

**Comment by ericonr** [comment]
Given comment in https://github.com/void-linux/void-packages/pull/28201#issuecomment-768477400 , I will be closing this. Feel free to reopen or open a new one.

**Pull request closed without merging**